packages/react-core/src/components/LoginPage/LoginForm.tsx

```diff
@@ -32,6 +35,12 @@ export interface LoginFormProps extends React.HTMLProps<HTMLFormElement> {
   onChangePassword?: (value: string, event: React.FormEvent<HTMLInputElement>) => void;
   /** Flag indicating if the Password is valid */
   isValidPassword?: boolean;
+  /** Flag indicating if the user can toggle hiding the password */
+  isShowPasswordEnabled?: boolean;
+  /** Label for the show password button */
+  showPasswordLabel?: string;
```

Comment:
It looks like this prop and the one below are used for aria-label.  If that is the case I suggest `showPasswordAriaLabel` and `hidePasswordAriaLabel